apps/prairielearn/src/lib/opsbot.js

```diff
@@ -58,7 +64,9 @@ export async function sendSlackMessage(
   });
 
   if (!response.ok) {
-    throw makeWithData(`Error sending message to ${channel}`, { body: await response.json() });
+    throw error.makeWithData(`Error sending message to ${channel}`, {
+      body: await response.json(),
```

Comment:
Nit: I'd be inclined to make both `sendSlackMessage` and `sendMessage` throw similarly-shaped errors.

Reply:
Done in d353d29.